vars/transportRequestCreate.groovy

```diff
@@ -11,20 +11,15 @@ import hudson.AbortException
 
 @Field def STEP_NAME = 'transportRequestCreate'
 
-@Field Set parameterKeys = [
-    'changeDocumentId',
-    'clientOpts',
-    'developmentSystemId',
-    'credentialsId',
-    'endpoint'
-  ]
-
 @Field Set stepConfigurationKeys = [
     'credentialsId',
     'clientOpts',
-    'endpoint'
+    'endpoint',
+    'developmentSystemId'
```

Comment:
'developmentSystemId' wasnt in **stepConfigurationKeys ** before

Reply:
Not fully sure how developmentSystemId will be used later on, maybe it makes sense to have it in the step config, maybe not. For now developmentSystemId is not added anymore.